README.md

```diff
@@ -159,38 +171,62 @@ function App() {
 export default App;
 ```
 
-You can pass Bridge options as a third parameter as well:
+`StoryblokComponent` here renders all the components dynamically which you loaded before during the initalization inside the `storyblokInit` function. 
+
+This is how you can pass the Bridge options as a third parameter to `useStoryblok` :
 
 ```js
-useStoryblok(story.id, (story) => (state.story = story), {
+useStoryblok(story.id, {version: 'draft'}, {
   resolveRelations: ["Article.author"],
 });
 ```
 
-#### 3. Link your components to Storyblok Visual Editor
+**Check out the React Bolierplate [here](https://github.com/storyblok/storyblok-react-boilerplate) or read on how to add Storyblok to React in 5 mins [here](https://www.storyblok.com/tp/headless-cms-react)**
 
-For every component you've defined in your Storyblok space, call the `storyblokEditable` function with the blok content:
 
-```js
-import { storyblokEditable } from "@storyblok/react";
+## Next 12 and Next 13 (no RSC)
 
-const Feature = ({ blok }) => {
-  return (
-    <div {...storyblokEditable(blok)} key={blok._uid} data-test="feature">
-      <div>
-        <div>{blok.name}</div>
-        <p>{blok.description}</p>
-      </div>
-    </div>
-  );
-};
+In this section, the way to use the SDK with the `pages` directory is described.
 
-export default Feature;
+The initalization remains the same when you work with Next.js. You can intialze the SDK in the `_app.js` file. Please refer to the 'Initialization' section above to read more.
+
+### Fetching Content
+
+The SDK provdies a `getStoryblokApi` object in your app, which is an instance of [storyblok-js-client](https://github.com/storyblok/storyblok-js-client). This can be used to fetch the content from Storyblok. You can use it in functions like `getStaticProps`, `getStaticPaths`, `getServerSideProps` etc.
```

Comment:
```suggestion
The SDK provides a `getStoryblokApi` object in your app, which is an instance of [storyblok-js-client](https://github.com/storyblok/storyblok-js-client). This can be used to fetch the content from Storyblok. You can use it in functions like `getStaticProps`, `getStaticPaths`, `getServerSideProps` etc.
```